bluesky/bundlers.py

```diff
@@ -40,9 +39,10 @@ def __init__(self, md, record_interruptions, emit, emit_sync, log, *, strict_pre
         self._config_ts_cache: ObjDict[Any] = dict()  # " obj.read_configuration() timestamps
         self._descriptors = dict()  # cache of {name: (objs_frozen_set, doc)}
         # cache of {obj: {objs_frozen_set: (name, uid)}}
-        self._local_descriptors: Dict[Any, Dict[FrozenSet[str], Tuple[str, str]]] = dict()
-        self._sequence_counters = dict()  # a seq_num counter per stream
-        self._teed_sequence_counters = dict()  # for if we redo data-points
+        self._local_descriptors: Dict[Any, Dict[FrozenSet[str], Tuple[str, str, Callable, Callable]]] = dict()
```

Comment:
Does this work?
```suggestion
        self._local_descriptors: Dict[Any, Dict[FrozenSet[str], ComposeDescriptorBundle] = dict()
```
Also for `self._descriptors` and others.

Reply:
ComposeDescriptorBundle doesn't contain the two strings in the tuple for values in `_local_descriptors` (for the stream_name and descriptor uid), similarly `_descriptors` contains an `obj_set` not present in the ComposeDescriptorBundle

Reply:
Should I rework the code to obtain that stuff from the doc and get rid of the current descriptor logic?

Reply:
If you think that would come out neater then yes

Reply:
`stream_name` should already be in the descriptor doc anyway (it's `name`), but I'm not convinced we should add `objs_read` into it, that's a bluesky thing rather than an event model thing... We could have a tuple of `DescriptorBundle` and `objs_read`?